src/main/java/org/team1540/robot2022/commands/intake/Intake.java

```diff
@@ -0,0 +1,31 @@
+package org.team1540.robot2022.commands.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.team1540.robot2022.Constants;
+
+public class Intake extends SubsystemBase {
+    private final Solenoid leftSolenoid = new Solenoid(PneumaticsModuleType.REVPH, Constants.IntakeConstants.leftSolenoidChannel);
+    private final Solenoid rightSolenoid = new Solenoid(PneumaticsModuleType.REVPH, Constants.IntakeConstants.rightSolenoidChannel);
+    private final TalonFX motor = new TalonFX(Constants.IntakeConstants.falcon);
+
+    public Intake() {
+        Constants.IntakeConstants.currentLimitConfig.applyTo(motor);
+    }
+
+    @Override
+    public void periodic() {
+    }
+
+    public void setTilt(Boolean state) {
```

Comment:
```suggestion
    public void setFolded(boolean state) {
```
`setTilt` implies to me that it's not a binary state
`boolean` vs `Boolean` again (https://stackoverflow.com/questions/3728616/boolean-vs-boolean-in-java)